.github/workflows/reuse.yaml

```diff
@@ -146,28 +146,28 @@ jobs:
         if: ${{inputs.tag == 'image' && steps.extract_branch.outputs.branch == 'main'}}
         env:
           COSIGN_EXPERIMENTAL: "true"
-          COSIGN_REPOSITORY: "ghcr.io/kyverno/signatures"
+          COSIGN_REPOSITORY: "ghcr.io/${{ github.repository_owner }}/signatures"
         run: |          
           cosign sign \
           -a "repo=${{ github.repository }}" \
           -a "workflow=${{ github.workflow }}" \
           -a "ref=${{ github.sha }}" \
-          ghcr.io/kyverno/${{inputs.image_name}}@sha256:${{ steps.get-step-image.outputs.digest }}
+          ghcr.io/${{ github.repository_owner }}/${{inputs.image_name}}@sha256:${{ steps.get-step-image.outputs.digest }}
```

Comment:
We should also make similar changes in the makefile.

Reply:
@prateekpandey14 - can you please check if we need to update this?

Reply:
Sure i will check